diff --git a/device/google/wahoo/gps.conf b/device/google/wahoo/gps.conf
index 54482b19..4fe74aad 100644
--- a/device/google/wahoo/gps.conf
+++ b/device/google/wahoo/gps.conf
@@ -54,6 +54,9 @@ CAPABILITIES=0x37
 # of b1000001010 makes 3 slots available
 # and the remaining 7 slots unwritable.
 # AGPS_CERT_WRITABLE_MASK=0
+SUPL_HOST = 127.0.0.1
+SUPL_PORT = 9999
+SUPL_MODE = 0
 
 ################################
 # EXTRA SETTINGS
@@ -95,7 +98,7 @@ LPPE_UP_TECHNOLOGY = 0
 #     GPS_LOCK
 # 0: disable
 # 1: enable
-AGPS_CONFIG_INJECT = 1
+AGPS_CONFIG_INJECT = 0
 
 # AP Coarse Timestamp Uncertainty
 ##################################################
